android/src/main/java/nordpol/TagDispatcher.java

```diff
@@ -10,68 +10,76 @@
 import android.content.IntentFilter;
 import android.os.Build;
 import android.os.Bundle;
+import android.widget.Toast;
 
 public class TagDispatcher {
     private static final int DELAY_PRESENCE = 5000;
     
     private OnDiscoveredTagListener tagDiscoveredListener;
-    private OnNfcDisabledListener nfcDisabledListener;
+    private boolean handleUnavailableNfc;
     private Activity activity;
 
+    public enum NfcStatus {
```

Comment:
@Franzaine If we do not return status for disabled, then the AVAILABLE state is not required and I think we should remove it.